src/open_samus_returns_rando/specific_patches/chozo_seal_patches.py

```diff
@@ -165,8 +166,9 @@ def update_item_seals(editor:PatcherEditor):
             scenario.raw.actors[16][seal]["type"] = "chozoseal"
 
         # Dependencies
-        for asset in editor.get_asset_names_in_folder("actors/props/chozoseal"):
-            editor.ensure_present_in_scenario(scenario_name, asset)
+        for level_pkg in editor.get_all_level_pkgs():
+            for asset in editor.get_asset_names_in_folder("actors/props/chozoseal"):
+                editor.ensure_present(get_package_name(level_pkg, asset), asset)
```

Comment:
The thing this PR is changing, is that it respects the vanilla distribution of files between the normal pkg and `_discardables.pkg`.

If that's the issue here, it should be fixed in a more common way or, at least, should also take care of the other code parts where it is done wrongly. E.g. the above method `add_chozo_seals` does the same and is just in sight here. I'm very sure that there are more parts.

A more common way would be preferred because I assume that this was also the culprit with Ridley.

Reply:
Should we change all instances of `ensure_present_in_scenario` to `ensure_present`?

What do you mean by common way?

Reply:
From my point of view, you have shown that we have a bug in `ensure_present_in_scenario` because it allows you to add a file to the wrong package.
I think the best would be to actually revert the changes here and just add the `get_package_name` call in `ensure_present_in_scenario`, so it can fixed in all places?!

I'm curious if ridley will work then with a chozo seal in surfaceb